lib/link.ts

```diff
@@ -4,19 +4,31 @@
 import * as log from "./log";
 import {
   link, LinkOptions, AmqpError, Dictionary, Source, TerminusOptions, SenderEvents, ReceiverEvents,
-  EventContext as RheaEventContext
+  EventContext as RheaEventContext, ConnectionEvents
 } from "rhea";
 import { Session } from "./session";
 import { Connection } from "./connection";
 import { Func, emitEvent, EmitParameters } from './util/utils';
 import { Entity } from "./entity";
-import { OperationTimeoutError } from "./operationTimeoutError";
+import { OperationTimeoutError } from "./errorDefinitions";
 
 export enum LinkType {
   sender = "sender",
   receiver = "receiver"
 }
 
+/**
+ * @interface LinkCloseOptions
+ * Describes the options that can be provided while closing the link.
+ */
+export interface LinkCloseOptions {
+  /**
+   * Indicates whether the underlying amqp session should also be closed when the
+   * link is being closed.
```

Comment:
Since the default is `true` which is against the usual boolean default of false, should we document that here as well?

Reply:
Done.